requirements.txt

```
@@ -7,3 +7,4 @@ setuptools_scm[toml]>=6.2
 torch==2.1.0+cpu
 accelerate
 optimum-intel
+neural_speed
```

Current we don't have pypi package published, how about install from source?

```suggestion
git+https://github.com/intel/neural-speed.git
```

cc @VincyZhang